Shared/Samples/Show mobile map package expiration date/ShowMobileMapPackageExpirationDateView.swift

```diff
@@ -0,0 +1,72 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ShowMobileMapPackageExpirationDateView: View {
+    /// A map with no specified style.
+    @State private var map = Map()
+    
+    /// The mobile map package.
+    @State private var mapPackage: MobileMapPackage?
+    
+    /// A Boolean value that indicates whether to show an error alert.
+    @State private var isShowingErrorAlert = false
+    
+    /// The error shown in the error alert.
+    @State private var error: Error? {
+        didSet { isShowingErrorAlert = error != nil }
+    }
+    
+    var body: some View {
+        ZStack {
+            MapView(map: map)
+                .task {
+                    do {
+                        // Load a local mobile map package from a URL.
```

Comment:
```suggestion
                        // Loads a local mobile map package from a URL.
```
I think the samples have standardized on this verb tense.

Reply:
I agree with @CalebRas. I think we discussed that internal comments don't have any standard other than it would be nice if they started with a capital letter and ended with a period.

Reply:
Ah, okay, thanks for straighten that out.